maps/donut3.dmm

```diff
@@ -20167,7 +20214,10 @@
 /turf/simulated/floor/wood/five,
 /area/station/hydroponics/bay)
 "fDr" = (
-/obj/machinery/cell_charger,
+/obj/machinery/cell_charger{
+	pixel_y = 6;
+	pixel_x = -1
+	},
```

Comment:
A good change, but mention it in the future please. Had to squint at the map renders to see it.

Reply:
update to master should have resolved